tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>Were you planning to add more disturbing signals until the model cracks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>I did, but the output figure weren't stored apparently. Fixed now.</p>